app/javascript/components/public_health/actions/UpdateCaseStatus.js

```diff
@@ -167,7 +164,7 @@ class UpdateCaseStatus extends React.Component {
                   type="switch"
                   id="apply_to_group"
                   label="Apply this change to the entire household that these monitorees are responsible for, if it applies."
-                  checked={this.state.apply_to_group}
+                  checked={this.state.apply_to_group === true || false}
```

Comment:
Why the check for explicit true/false here?

Reply:
oops, I don't remember changing this line of code back to the way it was before (https://github.com/SaraAlert/SaraAlert/blame/e88fb22fd9698d3c6c5d2b51488669cb715dbcc9/app/javascript/components/publicHealth/actions/UpdateCaseStatus.js#L196) and not sure how this happened, but thanks for the catch!